packages/web/src/components/pool/my-liquidity-content/MyLiquidityContent.tsx

```diff
@@ -91,7 +91,7 @@ const MyLiquidityContent: React.FC<MyLiquidityContentProps> = ({
       return "$0";
     }
     const balance = Object.values(allBalances).reduce((acc, current) => acc += current.balanceUSD, 0);
-    return toLowerUnitFormat(balance, true);
+    return formatUsdNumber(String(balance), undefined, true);
```

Comment:
Do not use `undefined` for optional parameters.

Reply:
Done. default = 2;